server/routes/api/protein.py

```diff
@@ -25,10 +30,293 @@
 @bp.route('/<path:dcid>')
 def get_node(dcid):
     """Returns data given a protein node."""
-    response = dc_service.fetch_data('/internal/bio', {
+    response = dc.fetch_data('/internal/bio', {
         'dcid': dcid,
     },
-                                     compress=False,
-                                     post=False,
-                                     has_payload=False)
+                             compress=False,
+                             post=False,
+                             has_payload=False)
     return response
+
+
+def id(id_or_dcid):
+    '''
+    Delete 'bio/' prefix from a biomedical DCID, if it is present.
+    E.g. 'bio/P53_HUMAN' --> 'P53_HUMAN'
+    '''
+    # .replace replaces all instances, but no biomedical DCID will contain more than once instance of 'bio/'
+    return id_or_dcid.replace(BIO_DCID_PREFIX, '')
+
+
+def dcid(id_or_dcid):
+    '''
+    Add 'bio/' prefix to an id if it does not already start with 'bio/'.
+    E.g. 'P53_HUMAN' --> 'bio/P53_HUMAN'
+    '''
+    if id_or_dcid.startswith(BIO_DCID_PREFIX):
+        return id_or_dcid
+    return BIO_DCID_PREFIX + id_or_dcid
+
+
+def node(protein_id_or_dcid, depth):
```

Comment:
This also looks like an internal function only so it should be _node(). Otherwise, you should have asserts/checks for the calls to id() and .split() not failing.

Reply:
Thanks for noticing I'll add these checks!

Reply:
I added an assert for in id() for input not having multiple instances of 'bio/', but I feel like the only (easy to implement) constraint for the split is that it's length 2, and `protein, species = node_id.split('_')` will already raise an error in this case.  Should I still add an explicit assert here?

Reply:
Looks good!